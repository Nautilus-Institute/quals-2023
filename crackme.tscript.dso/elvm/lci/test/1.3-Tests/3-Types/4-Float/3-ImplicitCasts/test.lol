HAI 1.3
	I HAS A var1 ITZ FAIL
	I HAS A var2 ITZ WIN
	I HAS A var3 ITZ 123
	I HAS A var4 ITZ "123"
	VISIBLE SUM OF 0.0 AN var1
	VISIBLE SUM OF 0.0 AN var2
	VISIBLE SUM OF 0.0 AN var3
	VISIBLE SUM OF 0.0 AN var4
KTHXBYE
