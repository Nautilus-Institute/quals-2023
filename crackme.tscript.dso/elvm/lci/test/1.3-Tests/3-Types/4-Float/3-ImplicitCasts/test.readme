This test checks to make sure other types are cast to floating point decimals
correctly.
