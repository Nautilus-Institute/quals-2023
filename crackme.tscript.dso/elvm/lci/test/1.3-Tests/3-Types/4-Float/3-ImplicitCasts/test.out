0.00
1.00
123.00
123.00
